mac_build/buildMacBOINC-CI.sh

```diff
@@ -257,3 +257,30 @@ fi
 echo "Building ${target}...done"
 
 cd ..
+
+set -e
+ 
+echo "Testing libs and binaries for univaral build."
+echo ""
+
+binaries_libs_list="*.a boinc switcher gfx_switcher boincscr setprojectgrp boinccmd SetUpSecurity AddRemoveUser BOINCManager PostInstall BOINCSaver *Installer Uninstall*"
+folders_list="3rdParty/buildCache/mac/lib mac_build/build/Deployment"
+
+for object in $binaries_libs_list; do
```

Comment:
Please move this check to the corresponding block above and test only those binaries that are build in those block. We don't need to wait to build everything if any of the targets was build with the wrong architecture.
Please do this as I suggested you before.

Reply:
I don't have a mac to understand which target belong to which binary. Also on android we test after all the binary are build. Mac when cache, is build around 5 min. So waiting for results for that, that ok. 
There never wasn't a test before, and now you sudden rush as immediate if the binary are wrong to fail the CI?
I will not invest more time for that. I am doing more meaningful tasks.

Reply:
I didn't ask you to add this check. You decided to do this by yourself. And telling me now that you don't have Mac and you can't check smth is not an excuse to leave the code in a non-appropriate state. From the very beginning I told you have I want to see this functionality, so don't blame me that you decided to ignore me and do it your own way.
If you don't want to make it properly - leave it, maybe one day I will find some time to make it properly.
I can't approve this PR in its current state.

Reply:
When I do something here because you told me?
I am doing stuff here because I care, because I want to change, and it for common good, and not between you and me.
The test is working, and Charlie already approve it. You are very good to preserve your own time, but want to waste other time. You can accept this code, and "fix" it when you "have time". 
Don't hold it up, and don't waste my time like you don't want to waste yours.

Reply:
Please remove wait for author respond. I already told my respond.

Reply:
At first, calm down. 
Second, I don't waste time of anyone. You don't want to work on smth - nobody forces you to do that. But if you started smth - please be patient and finish your work.
Third, that fact that you care about smth doesn't mean that you should produce unfinished or incorrect code. In this case improvement is required to make this code looks nice and professional. If you're professional - you should listen to the other people and hear their arguments: "I don't have a mac" and "I am doing more meaningful tasks." are not valid arguments from your side to prove that your opinion is correct.
I gave my arguments: checking built binaries right after they are built is way more efficient than waiting for all of them to build and then try to understand which target was not build correctly.
If you don't agree and don't want to proceed - fine, leave it and I'll finish it when I have time for this.

Reply:
> Please remove wait for author respond. I already told my respond.

I made a new label specially for you.